### PR TITLE
Replace http with https

### PR DESCRIPTION
**Pull request by Falke-Design**

### CODE REVIEW

**Review by IvanSanchez — APPROVED** [review]

File: src/geometry/LineUtil.js [hunk 1]
```diff
@@ -14,11 +14,11 @@ import * as Util from '../core/Util';
 // @function simplify(points: Point[], tolerance: Number): Point[]
 // Dramatically reduces the number of points in a polyline while retaining
 // its shape and returns a new array of simplified points, using the
-// [Douglas-Peucker algorithm](http://en.wikipedia.org/wiki/Douglas-Peucker_algorithm).
+// [Douglas-Peucker algorithm](https://en.wikipedia.org/wiki/Douglas-Peucker_algorithm).
```
Comment (jonkoops) [inline src/geometry/LineUtil.js:17]: This link now redirects to another page, let's update it.
```suggestion
// [Ramer–Douglas-Peucker algorithm](https://en.wikipedia.org/wiki/Ramer–Douglas–Peucker_algorithm).

```

File: src/geometry/LineUtil.js [hunk 2]
```diff
@@ -47,7 +47,7 @@ export function closestPointOnSegment(p, p1, p2) {
 	return _sqClosestPointOnSegment(p, p1, p2);
 }
 
-// Douglas-Peucker simplification, see http://en.wikipedia.org/wiki/Douglas-Peucker_algorithm
+// Douglas-Peucker simplification, see https://en.wikipedia.org/wiki/Douglas-Peucker_algorithm
```
Comment (jonkoops) [inline src/geometry/LineUtil.js:50]: Same here.
```suggestion
// Ramer–Douglas-Peucker simplification, see https://en.wikipedia.org/wiki/Ramer–Douglas–Peucker_algorithm
```

File: src/layer/vector/Canvas.js [hunk 3]
```diff
@@ -13,7 +13,7 @@ import {Bounds} from '../../geometry/Bounds';
  * Allows vector layers to be displayed with [`<canvas>`](https://developer.mozilla.org/docs/Web/API/Canvas_API).
  * Inherits `Renderer`.
  *
- * Due to [technical limitations](http://caniuse.com/#search=canvas), Canvas is not
+ * Due to [technical limitations](https://caniuse.com/#search=canvas), Canvas is not
```
Comment (jonkoops) [inline src/layer/vector/Canvas.js:16]: We can use a direct link to the feature here.
```suggestion
 * Due to [technical limitations](https://caniuse.com/canvas), Canvas is not
```

File: src/layer/vector/SVG.js [hunk 4]
```diff
@@ -17,7 +17,7 @@ export var create = Browser.vml ? vmlCreate : svgCreate;
  * Allows vector layers to be displayed with [SVG](https://developer.mozilla.org/docs/Web/SVG).
  * Inherits `Renderer`.
  *
- * Due to [technical limitations](http://caniuse.com/#search=svg), SVG is not
+ * Due to [technical limitations](https://caniuse.com/#search=svg), SVG is not
```
Comment (jonkoops) [inline src/layer/vector/SVG.js:20]: We can use a direct link to the feature here.
```suggestion
 * Due to [technical limitations](https://caniuse.com/svg), SVG is not
```

File: src/map/Map.methodOptions.leafdoc [hunk 5]
```diff
@@ -31,7 +31,7 @@ Maximum age of detected location. If less than this amount of milliseconds
 passed since last geolocation response, `locate` will return a cached location.
 
 @option enableHighAccuracy: Boolean = false
-Enables high accuracy, see [description in the W3C spec](http://dev.w3.org/geo/api/spec-source.html#high-accuracy).
+Enables high accuracy, see [description in the W3C spec](https://dev.w3.org/geo/api/spec-source.html#high-accuracy).
```
Comment (jonkoops) [inline src/map/Map.methodOptions.leafdoc:34]: The hash in this url does not point to any specific header on the page, perhaps it has changed?

**Review by jonkoops — CHANGES_REQUESTED** [review]

File: src/map/Map.methodOptions.leafdoc (hunk 5 above)
Comment (johnd0e) [reply src/map/Map.methodOptions.leafdoc:34]: https://w3c.github.io/geolocation-api/#enablehighaccuracy-member

File: src/geometry/LineUtil.js [hunk 6]
```diff
@@ -14,11 +14,11 @@ import * as Util from '../core/Util';
 // @function simplify(points: Point[], tolerance: Number): Point[]
 // Dramatically reduces the number of points in a polyline while retaining
 // its shape and returns a new array of simplified points, using the
-// [Douglas-Peucker algorithm](http://en.wikipedia.org/wiki/Douglas-Peucker_algorithm).
+// [Ramer-Douglas-Peucker algorithm](https://en.wikipedia.org/wiki/Ramer-Douglas-Peucker_algorithm).
 // Used for a huge performance boost when processing/displaying Leaflet polylines for
 // each zoom level and also reducing visual noise. tolerance affects the amount of
 // simplification (lesser value means higher quality but slower and with more points).
-// Also released as a separated micro-library [Simplify.js](http://mourner.github.com/simplify-js/).
+// Also released as a separated micro-library [Simplify.js](https://github.com/mourner/simplify-js/).
```
Comment (mourner) [inline src/geometry/LineUtil.js:21]:
```suggestion
// Also released as a separated micro-library [Simplify.js](https://mourner.github.io/simplify-js/).
```

**Review by mourner — APPROVED** [review]

**Review by jonkoops — APPROVED** [review]

**Review by swansoffie01 — COMMENTED** [review]
c33f018

**Pull request merged**